corehq/apps/translations/app_translations/utils.py

```diff
@@ -129,6 +129,19 @@ def is_single_sheet(identifier):
     return identifier == SINGLE_SHEET_NAME
 
 
+def is_single_sheet_workbook(workbook):
+    sheets_count = len(workbook.worksheets)
+
+    if sheets_count != 1:
+        return False
+
+    first_sheet = workbook.worksheets[0]
+    if is_single_sheet(first_sheet.title):
+        return True
+
+    return False
+
+
```

Comment:
nit: i'd prefer how this was written earlier, returning `True` for what it should be instead of now returning `False` for what it should not be. Overall it was just easier to understand and short.

Reply:
Updated in 12504db8f96, is that what you meant?

Reply:
yes